src/modules/fixedwing_backside/fixedwing.cpp

```diff
@@ -142,7 +142,7 @@ void BlockMultiModeBacksideAutopilot::update()
 
 	// store old position command before update if new command sent
 	if (_missionCmd.updated()) {
-		_lastMissionCmd = _missionCmd.getData();
```

Comment:
Do you realistically have time to maintain this? Because if not it might make sense to remove this (I do realise it has been hard work) from the repo for the sake of focusing efforts on what we can maintain.